src/app/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -1,15 +1,36 @@
 import React, { useContext } from 'react';
 import { createInstance, OptimizelyProvider } from '@optimizely/react-sdk';
+import { v4 as uuid } from 'uuid';
+import Cookie from 'js-cookie';
 import { ServiceContext } from '#contexts/ServiceContext';
-import { getAtUserId } from '#lib/analyticsUtils';
+import onClient from '#lib/utilities/onClient';
+import isOperaProxy from '#lib/utilities/isOperaProxy';
+
+const getCknsMvt = () => {
+  // Users accessing the site on opera "extreme data saving mode" have the pages rendered by an intermediate service
+  // Attempting to track these users is just tracking that proxy, causing all opera mini visitors to have the same id
+  if (!onClient() || isOperaProxy()) return null;
+
+  const cookieName = 'ckns_mvt';
+  const cookieValue = Cookie.get(cookieName);
+  const expires = 365; // expires in 12 Months
+
+  if (!cookieValue) {
+    const cookieUuid = uuid();
+    Cookie.set(cookieName, uuid(), { expires, path: '/' });
```

Comment:
Does the value being a `uuid` align with other teams usages for the cookie?

Reply:
Yep, I won't link to the repos, but you can search the organisation for that cookie in code and they all use the uuid v4 standard